modules/MIPS/clinical_measures.php

```diff
@@ -332,7 +321,7 @@ function Form_Validate() {
                         onkeyup='datekeyup(this,mypcc)' onblur='dateblur(this,mypcc)'
                         title='<?php echo htmlspecialchars(xl('yyyy-mm-dd hh:mm:ss'), ENT_QUOTES); ?>'>
 <?php   if(empty($report_id)) { ?>
-                      <img src='../pic/show_calendar.gif' align='absbottom' width='24' height='22'
+                      <img src='../../interface/pic/show_calendar.gif' align='absbottom' width='24' height='22'
```

Comment:
Are you still using the old date picker? It is not a problem and it IS SOMETHING that can be changes after the code is brought in. Just needs to be on a HIT list.